### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mariahkannenberg**

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/ustc-story-template.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ assignees: ''
 
 ## Mobile Design/Considerations
 
+## IRS API Considerations
+Do these changes impact the IRS API?
```
Comment (mmarcotte) [inline .github/ISSUE_TEMPLATE/ustc-story-template.md:18]: I think I would rephrase:

```
- the IRS API
+ the V1 or V2 API endpoints (used by the IRS)
```
